src/game/client/components/chat.cpp

```diff
@@ -1309,16 +1314,28 @@ void CChat::OnRender()
 
 		if(pLine->m_Highlighted)
 		{
-			TextRender()->TextColor(TextColor);
-			TextRender()->TextOutlineColor(ColorHighlightOutline);
-			TextRender()->TextEx(&Cursor, pLine->m_aText, -1);
+			TextRender()->TextColor(TextColor.r, TextColor.g, TextColor.b, TextColor.a);
```

Comment:
The other TextColor and TextSecondaryColor methods can be used to to pass the vec4 directly.

The call to TextColor appears first in both if branches, so it can be pulled up.